src/plugins/ui/selection_input.ts

```diff
@@ -76,6 +81,11 @@ export class SelectionInputPlugin extends UIPlugin implements StreamCallbacks<Se
           return CommandResult.MaximumRangesReached;
         }
         break;
+      case "CHANGE_RANGE":
```

Comment:
From a functional POV, what happens in the side panel looks strange when you paste something like `A1,B2`. You press confirm, and there no error message, but the input is red, and its back at the previous value

Reply:
ok  I changed the behavior - if you input a set or ranges 'A1,A2,...' it will take the first element and drop the rest of the string